libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2359,17 +2359,12 @@ bool WaveTrack::FormatConsistencyCheck() const
       });
 }
 
-bool WaveTrack::InsertClip(WaveClipHolder clip)
+void WaveTrack::InsertClip(WaveClipHolder clip)
 {
-   if(!clip->GetIsPlaceholder() && clip->IsEmpty())
```

Comment:
I think that the explanation of this line presense is that we've found out that zero clips could have been created during editing the project.